kfs-core/src/main/java/edu/arizona/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImpl.java

```diff
@@ -118,13 +145,13 @@ public boolean rerouteProcurementCardDocuments() {
               //step two, reroute documents enroute to the reconciler group(s) that had group membership problems
                 else if (node.equals("ProcurementCardReconciler") && groupRerouting && 
                         parameterService.getParameter(ProcurementCardRerouteDocumentsStep.class,
-                                RECONCILER_GROUPS_TO_REROUTE_PARM_NM).getValue() != null
+                                KFSParameterKeyConstants.ProcurementCardParameterConstants.RECONCILER_GROUPS_TO_REROUTE_PARM_NM).getValue() != null
```

Comment:
This method is outside of the scope of the Step 12 ticket, and is part of Step 13-16. This was only modified due to moving the constants to where they belong.